turbopack/crates/turbopack-core/src/issue/analyze.rs

```diff
@@ -11,10 +11,10 @@ use crate::ident::AssetIdent;
 
 #[turbo_tasks::value(shared)]
 pub struct AnalyzeIssue {
-    pub severity: Vc<IssueSeverity>,
+    pub severity: ResolvedVc<IssueSeverity>,
     pub source_ident: Vc<AssetIdent>,
```

Comment:
I couldn't use `ResolvedVc<T>` here because of `emit` function implementation in the file.